pkgs/games/steam/steam.nix

```diff
@@ -2,15 +2,15 @@
 
 let
   traceLog = "/tmp/steam-trace-dependencies.log";
-  version = "1.0.0.69";
+  version = "1.0.0.70";
 
 in stdenv.mkDerivation {
   pname = "steam-original";
   inherit version;
 
   src = fetchurl {
     url = "https://repo.steampowered.com/steam/pool/steam/s/steam/steam_${version}.tar.gz";
```

Comment:
We could use urls and upload it to web.archive.org :P

Reply:
Steam keeps an archive of all stable releases at <https://repo.steampowered.com/steam/archive/stable/>. They keep an archive off **all** releases at <https://repo.steampowered.com/steam/archive/precise/>. The stable archive is recommended. The `pool/` URL only contains the latest stable and beta, we shouldn't use those URLs, as they are frequently removed. You can read about all of this at <https://repo.steampowered.com/steam/>.

Reply:
@SuperSandro2000 @delroth 
Can the change to the URL also be done in the current PR?
If so, it would need two commits (imo): one for the version change and the other for the URL change.

Reply:
Yes, please. You can throw it into the version update.

Reply:
I applied this in #120545